### PR TITLE
 Ajay PR T2.1 Gif Images Updates

### PR DESCRIPTION
**Pull request by Mohammed-MSFT**
1) bot-conversation-sso-quickstart
2) bot-file-upload/python
3) bot-task-module
4) bot-teams-authentication/python
5) meeting-tabs/csharp
6) meetings-attendance-report
7) meeting-notification/csharp
8) msgext-action-quickstart/js
9) msgext-link-unfurling/python
10) msgext-search/python
11) tab-adaptive-cards/csharp
12) tab-personal/razor-csharp
13) tab-request-approval/csharp

### CODE REVIEW

**Review by Harikrishnan-MSFT — APPROVED** [review]
Looks good approved

File: samples/bot-teams-authentication/python/Images/4.User_Token.png
Comment (Pawank-MSFT) [inline samples/bot-teams-authentication/python/Images/4.User_Token.png:1]: update Token image and hide in black line.

File: samples/tab-adaptive-cards/csharp/TabWithAdpativeCardFlow/Images/3-AccountSelection.png
Comment (Pawank-MSFT) [inline samples/tab-adaptive-cards/csharp/TabWithAdpativeCardFlow/Images/3-AccountSelection.png:1]: hide emails

**Review by Pawank-MSFT — COMMENTED** [review]
some comments plz fix

File: samples/bot-task-module/nodejs/Images/10.Powerapp_in_Tab.png
[the remark this answers is not in the document]
Comment (Mohammed-MSFT) [reply samples/bot-task-module/nodejs/Images/10.Powerapp_in_Tab.png:1]: Fixed image, and gif file as well.

File: samples/tab-adaptive-cards/csharp/TabWithAdpativeCardFlow/Images/3-AccountSelection.png
Comment (Mohammed-MSFT) [reply samples/tab-adaptive-cards/csharp/TabWithAdpativeCardFlow/Images/3-AccountSelection.png:1]: Image Fixed and Gif File.

File: samples/bot-teams-authentication/python/Images/4.User_Token.png
Comment (Mohammed-MSFT) [reply samples/bot-teams-authentication/python/Images/4.User_Token.png:1]: Fixed Image and gif file

**Review by Pawank-MSFT — APPROVED** [review]
looks correct, Approving!

**Comment by Pawank-MSFT** [comment]
Executed build pipeline: https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/10237073207

**Pull request merged**